src/pages/home/report/ReportActionItem.js

```diff
@@ -142,6 +142,7 @@ class ReportActionItem extends Component {
             children = (
                 <IOUAction
                     chatReportID={this.props.report.reportID}
+                    iouReportID={this.props.report.iouReportID}
```

Comment:
`iouReportID` passed but not defined in the prop types?

Reply:
It's only used in `compose`, if I define it here eslint will complain about it being unused.